README.md

```diff
@@ -188,6 +188,26 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+### Conditionally including or excluding attributes
+
+You can also dynamically include or exclude serialized attributes by defining a
+method named include_attributes. This is typically used to customize the output
+based on `current_user`. For example:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  attributes :id, :body, :author
+
+  def include_attributes(included)
+    if current_user.admin?
+      included
+    else
+      included - [:author]
+    end
+  end
```

Comment:
With the OO approach, how could that be easily used with `has_many`? Is it possible to change the serializer for relationships dynamically?